nncf/quantization/metrics.py

```diff
@@ -21,6 +22,8 @@ def get_metric_table(self):
         pass
 
 
+# pylint:disable=protected-access
+# pylint:disable=unused-variable
```

Comment:
Wwhere were this two warning reported? Is it really False Positive?

Reply:
This is not my code, so decided not delete unused variables, perhaps they are imported in other files. 
Pylint mark variables on 255, 361 and 498 lines as unusable.
Protected-access warnings are in 295, 362, 365, 419, 429, 458, 480 lines

Reply:
please remove these variables, find author by git blame and add them to review.
AFAIK, @asenina was working on this feature.

Reply:
@pfinashx  Please take changes for `nncf\quantization\metrics.py` on this [branch](https://github.com/asenina/nncf_pytorch/tree/as/fix_pylint) . Thanks.

Reply:
@asenina Done, thanks